### PR TITLE
Get rid of npm-shrinkwrap

### PR DESCRIPTION
**Pull request by yusefnapora**
So, I got bit by a bunch of shrinkwrap-related bugs today, many of which were just oversights on my part:

- I added a dependency in #117 but didn't update the shrinkwrap; installing without a cached `node_modules` caused `build-jq.js` to fail
- I tried to fix it by pushing an updated `npm-shrinkwrap.json` straight to master (sad, I know).  However, this morning I was messing about with node and npm versions trying to track down a bug in js-libp2p-crypto, so I had npm 4 instead of npm 3 installed.  
- It turns out that npm 3 (the default on travis for node 6) fails to install aleph on linux with a shrinkwrap file generated by npm 3 on a mac.  This is because we have a platform-specific dependency on the `fsevents` module, and while a regular `npm install` is smart enough to detect the platform mismatch, for some reason the two npm versions differ enough that npm 3 tries to build the mac-only package on linux because it's in the shrinkwrap file generated by npm 4.

In conclusion: 😱 😡 🔫 

If it weren't for the npm 3 vs 4 issue, I'd be more inclined to try to automate the shrinkwrap generation so I don't make the original mistake of forgetting to update it.  But it seems crazy to only be able to support one version or the other, especially since both are in pretty wide use.

[yarn](https://github.com/yarnpkg/yarn) is pretty great, but idk if we want to make end users install it... we could make it "optional, unless npm is being stupid, then give it a try" 😄

### CODE REVIEW

**Comment by parkan** [comment]
Let's keep shrinkwrap and do 3->4 only
